app/models/comfy/cms/category.rb

```diff
@@ -1,6 +1,9 @@
 require ComfortableMexicanSofa::Engine.root.join('app', 'models', 'comfy', 'cms', 'category')
 
 class Comfy::Cms::Category < ActiveRecord::Base
+  validates_presence_of :label, :title_en, :title_cy
```

Comment:
You could stack these up a la `validate :label, presence: true, uniqueness: true #...` did you chose not to? I don't feel strongly about it either way, just wondering :+1: